packages/aws-lambda/test/lambdas/async.js

```diff
@@ -20,33 +20,37 @@ require('../../src/metrics/rootDir').root = require('path').resolve(__dirname, '
 const fetch = require('node-fetch');
 
 const downstreamDummyUrl = process.env.DOWNSTREAM_DUMMY_URL;
+const response = {};
 
-const response = {
-  headers: {
-    'X-Response-Header-1': 'response header value 1',
-    'X-Response-Header-3': 'response header value 3'
-  },
-  multiValueHeaders: {
-    'X-Response-Header-2': ['response', 'header', 'value 2'],
-    'X-Response-Header-4': ['response', 'header', 'value 4']
-  },
-  body: {
-    message: 'Stan says hi!'
+const handler = async event => {
+  console.log('in actual handler');
+
+  if (event.version === '1.0') {
```

Comment:
FYI: We could refactor and optimize the lambda handler types. But ignore that for now.